uswds/uswds
# USWDS - Indeterminate Checkbox: Cleanup scss and stories

# Summary

Runs prettier and reorganizes storybook twig files to match existing variant patterns.

## Breaking change

This is not a breaking change.

## Related issue

Resolves #5658 

## Related pull requests

#5549 
#5648 

## Preview link

[Indeterminate checkbox story →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-cleanup-scss-and-stories/?path=/story/components-form-inputs-checkbox--indeterminate)
[Indeterminate checkbox tile story →](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-cleanup-scss-and-stories/?path=/story/components-form-inputs-checkbox--indeterminate-tile)

## Problem statement

Contributed work needed some minor adjustments to match USWDS patterns.

## Solution

1. Run prettier
2. Create isolated indeterminate checkbox stories
3. Remove indeterminate examples from pre-existing stories.

## Major changes

Added indeterminate storybook control and stories to display the new variant.

>[!NOTE]
> After discussion, we decided this qualified as a variant and _not_ a test.

## Testing and review

1. Confirm there are no build errors.
2. Confirm running `npm run lint:sass` does not flag any failures.
3. Confirm `npm run prettier` does not introduce any changes.
5. Existing checkbox stories matches develop.
6. Confirm test stories match the existing pattern for test features.
